lib/matplotlib/font_manager.py

```diff
@@ -656,6 +656,7 @@ def __init__(self,
             if (style is None and variant is None and weight is None and
                     stretch is None and size is None and fname is None):
                 self.set_fontconfig_pattern(family)
+                self.set_math_fontfamily(math_fontfamily)
```

Comment:
Since this line is executed both inside and outside the `if` (see lines 659 and 669) I believe we could just change line 651 to:

```py
self.set_math_fontfamily(math_fontfamily)
```

And remove lines 659 and 669.

Reply:
I'm not quite familiar with the matplotlib coding style. Do you think we should keep the claim sentence in `__init__` for better understanding?

```python3
self._math_fontfamily = None
self.set_math_fontfamily(math_fontfamily)
```
 Or simply remove the first line?

Reply:
You can remove the first line. I think there is a way to clearly show the suggestions you want someone to do, unfortunately I am not that versed into github yet. My apologies.

Reply:
I've modified as what you said.  

The suggestion button is just in the comment box, but not always useful if you want to make suggestion on some unchanged codes in a PR.  
![image](https://user-images.githubusercontent.com/13214530/116594460-3b926780-a8f0-11eb-8e53-18edab40ed19.png)

Reply:
I tryed this button, but just like you said, it wouldn't let me include the part of the code you didn't change. Unfortunate.